src/theme/mui.js

```diff
@@ -308,17 +310,31 @@ export default createMuiTheme({
     },
     MuiListItemText: {
       primary: {
-        fontFamily: 'Averta, monospace',
+        fontFamily: secondaryFontFamily,
         fontSize: mediumFontSize,
         fontWeight: bolderFont,
         color: primary,
       },
       secondary: {
-        fontFamily: 'Averta, monospace',
+        fontFamily: secondaryFontFamily,
         fontSize: smallFontSize,
         color: disabled,
       },
     },
+    MuiCheckbox: {
+      colorSecondary: {
+        '&$disabled': {
+          color: rgba(secondary, 0.5),
```

Comment:
rgba is undefined, commit https://github.com/gnosis/safe-react/pull/296/commits/fe422cba9f053a41b498df209ce19e46c69d8229 removed it

Reply:
Package `polish` was removed because it was incorrect, instead we should use `polished`

Reply:
Also please update the flow types after removing `polish` 😄

Reply:
This is weird, I don't know why there was a `polish` there (I suppose it was a typo). But Flow should've warned us (I guess) about the wrong import, and it didn't...

Maybe `flow-typed` overlooked it.

Anyways: I restored the import in `src/theme/mui.js`, added the correct library `"polished": "^3.4.2",`, and updated `flow-typed` files.

Thanks for your input.